joplin/api/schema.py.orig

```diff
@@ -0,0 +1,954 @@
+import django.utils.translation
+import graphene
```

Comment:
do we need to keep the old schema? there are merge conflicts to boot

Reply:
Wait, yeah I totally missed this. @briaguya whats the deal with this file?

Reply:
whoops didn't mean to check that in, good catch (too much merging)